scripts/assemble_changelog.py

```diff
@@ -214,6 +217,15 @@ def add_categories_from_text(self, filename, line_offset,
                                        line_offset + category.title_line,
                                        'Unknown category: "{}"',
                                        category.name.decode('utf8'))
+
+            body_split = category.body.splitlines()
+            for line in body_split:
+                if len(line) > MAX_LINE_LENGTH:
+                    raise InputFormatError(filename,
+                                           line_offset + category.title_line,
+                                           'Category body line too long: "{} ({})"',
+                                           category.name.decode('utf8'), len(line))
```

Comment:
The message would be easier to act on if it included the maximum length. Also I don't think “category body” is comprehensible out of context.
```suggestion
                                           'Line too long ({}, max {}) under "{}"',
                                           len(line), MAX_LINE_LENGTH, category.name.decode('utf8'))
```
It would be better to have a line number, but not worth the effort.